software/wikidocs.yml

```diff
@@ -0,0 +1,14 @@
+name: "WikiDocs"
+website_url: "http://wikidocs.it"
+source_code_url: "https://github.com/Zavy86/WikiDocs"
+description: "A databaseless markdown flat-file wiki engine."
+licenses:
+  - MIT
+platforms:
+  - PHP
+  - Docker
+# list of tags (categories), see https://github.com/awesome-selfhosted/awesome-selfhosted-data/tree/master/tags for the full list of tags
```

Comment:
Please remove this line

Reply:
Dang missed it. Sloppy on my part. Fixed.